docs/error-reporting/workflow-integrations/alerting-ops/opsgenie.md

```diff
@@ -19,19 +19,15 @@ The steps in this process are:
 In OpsGenie, click Integrations in the left navbar and click "Add" on "API". Note the API Key - you will need this in the next step - then modify the API integration as necessary, and click Save Integration.
 
 ## Set Up the Integration
-To set up the integration, first go to the Configuration page within the Web Console:
+To set up the integration, first go to the Project Settings page for the project you want to add a integration for:
 
-<img src={useBaseUrl('img/error-reporting/workflow-integrations/configure-org.png')} alt="" />
+<img src={useBaseUrl('img/error-reporting/workflow-integrations/set-up-workflow-integration.png')} alt="" />
 
-Next, select the project you want to add a integration for:
+Then click Integrations in the left-hand menu, and the plus sign to create a new integration:
```

Comment:
[Interface elements need to be bold](https://wiki.saucelabs.com/display/PM/Technical+Documentation+Style+Guide#TechnicalDocumentationStyleGuide-Bold,Italic,Underline)

```suggestion
Then click **Integrations** in the left-hand menu, and the plus sign to create a new integration:
```